src/dsp_tools/xmllib/value_converters.py

```diff
@@ -18,3 +21,57 @@ def convert_to_bool_string(value: Any) -> str:
     elif str_val in ("true", "1", "1.0", "yes"):
         return "true"
     return str(value)
+
+
+def replace_newlines_with_tags(text: str, converter_option: NewlineReplacementTags) -> str:
+    """
+    Converts the newlines in a string to XML tags.
+    The type of tags is specified through the converter_option enum.
+
+    Args:
+        text: string to convert
+        converter_option: tag options
+
+    Returns:
+        string with replaced values
+
+    Raises:
+        InputError: If an invalid conversion option is given
+    """
+    match converter_option:
+        case NewlineReplacementTags.none:
+            return text
+        case NewlineReplacementTags.newline:
+            return replace_newlines_with_br_tags(text)
+        case NewlineReplacementTags.paragraph:
+            return replace_newlines_with_paragraph_tags(text)
+        case _:
+            raise InputError(f"The conversion option for the text does not exist: {converter_option}.")
```

Comment:
I don't think this is even possible with enums?

Reply:
That is true, it was just habit. I removed it.